lib/tasks/dump.rake

```diff
@@ -242,13 +242,21 @@ namespace :db do
       %w[dev dev-lgfs staging api-sandbox production]
     end
 
-    def excluded_tables
+    def sensitive_tables
       %w(providers users claims defendants messages documents active_storage_blobs)
     end
 
     def sensitive_table_exclusions
       # make sure a db:dump task exists for each of the excluded tables (i.e. db:dump:providers)
-      excluded_tables.map { |table| "--exclude-table-data #{table}" }.join(' ')
+      sensitive_tables.map { |table| "--exclude-table-data #{table}" }.join(' ')
+    end
+
+    def unneeded_tables
+      %w(versions)
+    end
+
+    def unneeded_table_exclusions
+      unneeded_tables.map { |table| "--exclude-table-data #{table}" }.join(' ')
```

Comment:
You might consider replacing this and `sensitive_table_exlusions` with something like:

```ruby
    def exclusions_for(tables)
      tables.map { |table| "--exclude-table-data #{table}" }.join(' ')
    end
```